web/book/src/reference/stdlib/transforms/join.md

```diff
@@ -9,8 +9,10 @@ join side:{inner|left|right|full} table (condition)
 ## Parameters
 
 - `side` decides which rows to include, defaulting to `inner`.
-- Relation reference
-- A condition
+- `table` - a reference to a relation, possibly including an assignment, e.g.
+  `var= ...`
+- `condition` - a boolean condition
```

Comment:
```suggestion
- a reference to a relation, possibly including an assignment, e.g.
  `var= ...`
- a boolean condition
```

Only `side` is a named param, so removing the others as quoted names — is that reasonable?

Reply:
Ahah! I was aiming for visual consistency to call out/mention each of the parameters. Try this...

Reply:
But surely we do want to visually discriminate between `side` and the others?